vsdownload.py

```diff
@@ -723,6 +724,18 @@ def extractPackages(selected, cache, dest):
         else:
             print("Skipping unpacking of " + p["id"] + " of type " + type)
 
+def copyDependentAssemblies(app):
+    if not os.path.isfile(app + ".config"):
+        return
+    dest = os.path.dirname(app)
+    ns = "{urn:schemas-microsoft-com:asm.v1}"
+    configuration = ET.parse(app + ".config")
+    for codeBase in configuration.findall(f"./runtime/{ns}assemblyBinding/{ns}dependentAssembly/{ns}codeBase/[@href]"):
+        href = codeBase.attrib["href"].replace("\\", "/")
+        src = os.path.join(dest, href)
+        if os.path.isfile(src):
+            shutil.copy(src, dest)
```

Comment:
Can you add some comments/examples about which files end up copied, from where, in this case?

Reply:
I updated commit message to include the example OpenTelemetry.Api.dll.